src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Enums/InstanceEventType.cs

```diff
@@ -70,7 +70,14 @@ public enum InstanceEventType
         /// Instance process task ended event.
         /// </summary>
 #pragma warning disable SA1300 // Element should begin with upper-case letter
-        process_EndTask
+        process_EndTask,
+#pragma warning restore SA1300 // Element should begin with upper-case letter
+
+        /// <summary>
+        /// Instance process task abandon event.
+        /// </summary>
+#pragma warning disable SA1300 // Element should begin with upper-case letter
```

Comment:
could we disable this warning type for the whole file to avoid having to set it for each property?